cms/models/pagemodel.py

```diff
@@ -179,7 +181,7 @@ def move_page(self, target, position='first-child'):
                 if target.get_root().get_next_sibling().pk == public.get_root().pk:
                     target = target.publisher_public
                 else:
-                    Logger.warning('tree may need rebuilding: run manage.py cms fix-tree')
+                    logger.warning('tree may need rebuilding: run manage.py cms fix-tree')
```

Comment:
Unrelated bugfix. `Logger` doesn't have this method. Presumably there's no test for this path, but I have no idea where to begin writing one.